app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    files = command.split()
+    if len(files) != 3 or files[1] == files[2] or files[0] != "cp":
+        return
```

Comment:
better print some message or raise Exception  so that if something goes wrong, the user can understand it